Gemfile

```diff
@@ -6,4 +6,4 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'byebug'
-gem 'faraday', ENV['FARADAY_VERSION'] if ENV['FARADAY_VERSION'].to_s != ''
+gem 'faraday-net_http_persistent', '~> 2.0'
```

Comment:
issue: I think this should specified as a dependency in the Gemspec. Especially since it's a requirement for the rake tasks.

Reply:
moved!